### ISSUE TITLE
[Feature Request] Request individual albums via Lidarr

### ISSUE THREAD

**Issue by meysq** [thomst08/requestrr#55]
Hi there! Currently the functionality for Lidarr seems to download an artist’s entire library when all you want is one LP. Would it be possible to add a request for one single album?

**Comment by thomst08** [comment]
Hey there,
Awesome idea, this was something I wanted to do when I originally was making the feature, but due to the way lidarr handles albums (called releases on the api side) it could return a number of them and it was abit confusing as to which one was which.

Basically it needed abit more time to refine, this is something I want to include later :)

**Comment by RenNagasaki** [comment]
Just leaving a vote here. Would love albums being added.

**Comment by jioadsfu** [comment]
I agree 1000%. When I am chilling is a discord with my friends, I frequently do not want to download an artists entire catalog and just want an album or even a single song.

**Comment by tacoroumen** [comment]
<img width="426" height="347" alt="Image" src="https://github.com/user-attachments/assets/50c91e4c-c6d8-450b-853e-bd2e64cb90df" />

currently working on this

created a pull request for code #91
